src/main/java/org/jitsi/videobridge/LipSyncHack.java

```diff
@@ -204,52 +284,40 @@ void onRTPTranslatorWillWriteAudio(
             if (acceptedVideoSSRCs.contains(receiveVideoSSRC)
                 || injections.containsKey(receiveVideoSSRC))
             {
-                // No need to hack this SSRC.
+                // No need to hack this SSRC, as it's either already accepted
```

Comment:
ah ha, ok.  so this is basically saying "either we've already started forwarding real packets or we've already initialized the hack" and in both of those we don't need to initialize the hack.  think i got it, thanks george